imports/client/components/HuntListPage.tsx

```diff
@@ -462,7 +458,8 @@ const HuntModalForm = React.forwardRef((props: HuntModalFormProps, forwardedRef:
         <Col xs={9}>
           <DiscordChannelSelector
             id={`${idPrefix}puzzle-hooks-discord-channel`}
-            disable={disableForm}
+            guildId={guildId}
+            disable={disableForm || guildId === undefined}
```

Comment:
If these are disabled due to unset `guildId`, can we display some sort of information scent as to why, and what would need to be done to enable it?

Reply:
Hmm. Maybe the right behavior here is to hide these fields entirely and replace them with an `<Alert>` or something?